cypress/integration/data-import/view-all-filter.spec.js

```diff
@@ -0,0 +1,137 @@
+import DataImportViewAllPage from '../../support/fragments/data_import/dataImportViewAllPage';
+import fileManager from '../../support/utils/fileManager';
+import getRandomPostfix from '../../support/utils/stringTools';
+
+describe('ui-data-import: Filter the "View all" log screen', () => {
+  // Create unique name for a MARC file
+  const uniqueFileName = `test${getRandomPostfix()}.mrc`;
+  let query;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    // create dynamically file with given name in fixtures
+    fileManager.createFile(`cypress/fixtures/${uniqueFileName}`);
+
+    // remove generated test file from fixtures after uploading
+    cy.uploadFile(uniqueFileName);
```

Comment:
I think you need to upload several files to related checks. For example - upload with error and upload without error

Reply:
Hi @dmitriyfolio. `cy.uploadFile()` function can only be used to upload  empty files. I was not able to create `marc` files programmatically with valid content. Do you know any libraries to create content for this type of files.

Reply:
unfortunately no. Let's add one file with one record like cypress/fixtures/oneMarcBib.mrc into  https://github.com/folio-org/stripes-testing/pull/174/files . For example oneMarcHoldings.mrc

Reply:
Got it.

Reply:
and could you try to generate incorrect file based on existing correct marc file(may be read the content + remove/add something + create new temporary file related with special test). And in case of correct file - read template + generate new with unique name + play checks + delete generated correct file(as in you current test)

Reply:
Yes, I can do it.